runtime/ftplugin/qf.vim

```diff
@@ -14,3 +14,10 @@ let b:undo_ftplugin = "set stl<"
 
 " Display the command that produced the list in the quickfix window:
 setlocal stl=%t%{exists('w:quickfix_title')?\ '\ '.w:quickfix_title\ :\ ''}\ %=%-15(%l,%c%V%)\ %P
+
+autocmd Syntax <buffer>
+      \ if exists('w:quickfix_title') && w:quickfix_title =~# '^:\%(Man\|Help\) TOC$' |
+      \   syntax match qfHideTOC "^.*| " conceal containedin=ALL |
+      \   setlocal concealcursor=nvi |
+      \   setlocal conceallevel=2 |
```

Comment:
`concealcursor=nvi` and `conceallevel=2` are my goto settings.  No other reason.